sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -95,6 +97,18 @@
     public ServiceBusClientBuilder() {
     }
 
+    /**
+     * Sets various options on the client for example applicationId.
```

Comment:
Another JavaDoc to flesh out, especially before it gets replicated across all libraries.